src/Tasks/Microsoft.NET.Build.Tasks/ResolveRuntimePackAssets.cs

```diff
@@ -85,7 +85,9 @@ protected override void ExecuteCore()
 
         private void AddRuntimePackAssetsFromManifest(List<ITaskItem> runtimePackAssets, string runtimePackRoot,
             string runtimeListPath, ITaskItem runtimePack)
-        {            
+        {
+            var assetSubPaths = new HashSet<string>();
```

Comment:
Should this be OrdinalIgnoreCase?

Reply:
It should be; even if the file system is case-sensitive, we wouldn't ever want case-insensitive duplicates in the list.

I'll fix.